packages/twenty-server/src/engine/core-modules/billing/dtos/outputs/billing-metered-product-usage.output.ts

```diff
@@ -14,17 +14,11 @@ export class BillingMeteredProductUsageOutput {
   periodEnd: Date;
 
   @Field(() => Number)
-  usageQuantity: number;
+  usedCredits: number;
 
   @Field(() => Number)
-  freeTierQuantity: number;
-
-  @Field(() => Number)
-  freeTrialQuantity: number;
+  grantedCredits: number;
```

Comment:
these are breaking change and will break production during deploy. If this only breaks the billing page it's fine. If it break the whole workspace, let's be careful (we should try to build the FE with the new queries, launch the back with the previous version of the code, flush redis cache, and make sure workspace keeps working)